library/spdm_requester_lib/libspdm_req_set_certificate.c

```diff
@@ -0,0 +1,185 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2021-2022 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "internal/libspdm_requester_lib.h"
+
+#if LIBSPDM_ENABLE_SET_CERTIFICATE_CAP
+
+/**
+ * This function sends SET_CERTIFICATE
+ * to set certificate from the device.
+ *
+ * @param  context                      A pointer to the SPDM context.
+ * @param  slot_id                      The number of slot for the certificate chain.
+ * @param  cert_chain                   The pointer for the certificate chain to set.
+ * @param  cert_chain_size              The size of the certificate chain to set.
+ * @param  session_id                   Indicates if it is a secured message protected via SPDM session.
+ *                                      If session_id is NULL, it is a normal message.
+ *                                      If session_id is NOT NULL, it is a secured message.
+ *
+ * @retval RETURN_SUCCESS               The measurement is got successfully.
+ * @retval RETURN_DEVICE_ERROR          A device error occurs when communicates with the device.
+ * @retval RETURN_SECURITY_VIOLATION    Any verification fails.
+ **/
+libspdm_return_t libspdm_try_set_certificate(void *context, uint8_t slot_id,

```

Comment:
After talking with @jyao1 the expectation is that `cert_chain` will be formatted as defined in Table 28 of the SPDM specification. This needs to made clear to the integrator, that they need to provide the `length`, `Reserved`, `RootHash`, and `Certificates` fields.

Reply:
Yes, please add some description in the function header, such as 
the cert chain is a full SPDM certificate chain, including Length and Root Cert Hash.

Also, if possible, use same comment for libspdm_get_certificate().

Reply:
Thanks. I have added the comment in `libspdm_try_set_certificate` and `libspdm_set_certificate` function header. 
And we can add the comment for libspdm_get_certificate() in a new patch.